File: python/svix/models/event_bridge_config_out.py
# this file is @generated
import typing as t

from .common import BaseModel


class EventBridgeConfigOut(BaseModel):
    event_bus_name: str

    detail_type: str

    access_key_id: t.Optional[str] = None

    role_arn: t.Optional[str] = None

    external_id: t.Optional[str] = None

    region: str

File: kotlin/lib/src/main/kotlin/models/EventBridgeConfigOut.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class EventBridgeConfigOut(
    val eventBusName: String,
    val detailType: String,
    val accessKeyId: String,
    val region: String,
)
